src/Functional.CQS.AOP.MetricsCapturing/Functional.CQS.AOP.MetricsCapturing.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\Projects\Functional.CQS\src\Functional.CQS.AOP.MetricsCapturing\Functional.CQS.AOP.MetricsCapturing.xml</DocumentationFile>
```

Comment:
This project has more machine-specific paths that should be switched to relative paths.

Reply:
Not sure why Visual Studio is being awkward like that, but I'll switch these up.